interop/jvm/src/main/scala/scalaz/zio/interop/console.scala

```diff
@@ -0,0 +1,40 @@
+package scalaz
+package zio
+package interop
+
+import java.io.IOException
+
+import zio.{ console => c }
+
+trait ScalazConsole {
+
+  /**
+   * Prints the string representation of an object to the console.
+   */
+  def put[A](a: A)(implicit ev: Show[A]): IO[IOException, Unit] =
+    c.putStr(ev.shows(a))
+
+  /**
+   * Prints the string representation of an object to the console, including a newline character.
+   */
+  def putLn[A](a: A)(implicit ev: Show[A]): IO[IOException, Unit] =
+    c.putStrLn(ev.shows(a))
+}
+
+trait CatsConsole {
+  import cats.{ Show => CatsShow }
+
+  /**
+   * Prints the string representation of an object to the console.
+   */
+  def put[A](a: A)(implicit ev: CatsShow[A]): IO[IOException, Unit] =
+    c.putStr(ev.show(a))
+
+  /**
+   * Prints the string representation of an object to the console, including a newline character.
+   */
+  def putLn[A](a: A)(implicit ev: CatsShow[A]): IO[IOException, Unit] =
+    c.putStrLn(ev.show(a))
+}
+
+package object console extends ScalazConsole with CatsConsole
```

Comment:
A final suggestion I have would be to move the cats one into the cats interop package, and the zio one into the zio interop package. Right now you have to import them both.